openedx/core/djangoapps/site_configuration/middleware.py

```diff
@@ -53,3 +54,26 @@ def _set_cookie_wrapper(key, value='', max_age=None, expires=None, path='/', dom
             response.set_cookie = _set_cookie_wrapper
 
         return response
+
+
+class AuthorizationCheckMiddleware(object):
+    """
+    Middleware for checking the authorization of users.
```

Comment:
The first line in the multiple doc strings should have the trailing empty line.

Reply:
Thank you. I did not know this, but now I will try to observe.